dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -580,12 +583,12 @@ def test_profile(self):
                 'bin_edges': np.array([2.5, 5.0, 7.5, 10.0, 12.5]),
             },
             quantiles={
-                0: 3.75,
-                1: 5.0,
-                2: 10.0
+                0: 35 / 8,  # 75% between 2.5 and 5 (histogram based)
```

Comment:
3.75 was the midpoint between 2.5 and 5.0, however the value at 5.0 is technically 33% of the way. Hence, the quantile at 25% would be 75% of that bin